### PR TITLE
Update HTTP specs to point to httpwg; update exception list

### PR DESCRIPTION
**Pull request by Elchi3**
browser-specs now supports IETF/httpwg.org specifications! 🎉 

This PR updates spec_urls to the preferable httpwg host and updates the exception list given a lot of specs aren't in need of an exception anymore. For those that are still an exception, I've filed issues with browser-specs and updated the reasons why our exceptions are still needed.

cc @sideshowbarker

### CODE REVIEW

**Comment by Elchi3** [comment]
The upstream issues have been solved already. Will update this again, once we bumped browser-specs to version 2.1.

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
dang — sorry, I didn’t see the **not ready** label on this until after I’d already pushed the merge button…

**Comment by Elchi3** [comment]
Will do the remaining work in a new PR. no worries.

**Comment by Elchi3** [comment]
The follow-up is in https://github.com/mdn/browser-compat-data/pull/11412

**Pull request merged**
